boards/st/nucleo_h723zg/Kconfig.defconfig

```diff
@@ -10,6 +10,10 @@ if NETWORKING
 config NET_L2_ETHERNET
 	default y
 
+config PHY_INIT_PRIORITY
+	default 81
+	depends on NET_L2_ETHERNET && ETH_DRIVER
+
```

Comment:
Can't this be moved at SoC level?  Is there a specific reason to make it board specific ?

Reply:
Hi @erwango ,
yes, setting it at soc-level looks a better approach, done.